### PR TITLE
Optimize the C++ code after running code quality check (Style)

### PR DESCRIPTION
**Pull request by jeevithiesh**

### CODE REVIEW

File: goa/Classes/AppDelegate.cpp [hunk 1]
```diff
@@ -195,13 +195,13 @@ int AppDelegate::printExtendedSceneGraph(int fd, Node *node, int level)
     int total = 1;
     //    for(int i=0; i<level; ++i)
     //        Console::Utility::sendToConsole(fd, "-", 1);
-    float x = 0.0;
-    float y = 0.0;
+    //float x;
+    //float y;
```
Comment (ankitk20) [inline goa/Classes/AppDelegate.cpp:199]: Remove if not required

File: goa/Classes/AppDelegate.cpp [hunk 2]
```diff
@@ -195,13 +195,13 @@ int AppDelegate::printExtendedSceneGraph(int fd, Node *node, int level)
...
     if (node->getParent() != nullptr)
     {
         auto nodeInWorld = node->getParent()->convertToWorldSpace(node->getPosition());
-        x = nodeInWorld.x;
-        y = nodeInWorld.y;
+    //    x = nodeInWorld.x;
+    //    y = nodeInWorld.y;
```
Comment (ankitk20) [inline goa/Classes/AppDelegate.cpp:204]: Remove if not required

File: goa/Classes/alphamon/Alphamon.cpp [hunk 3]
```diff
@@ -239,8 +239,8 @@ void Alphamon::alphamonLegAnimation(std::string animationName, bool loop) {
 	for (auto item = children.rbegin(); item != children.rend(); ++item) {
 		Node * monsterItem = *item;
 		std::string str = monsterItem->getName().c_str();
-		if (str.find("skate") == 0) {
-			auto legTimeline = CSLoader::createTimeline(CCString::createWithFormat("leg_ani/%s.csb", str.c_str())->getCString());
+        if (str.compare(0, 5, "skate")){
```
Comment (ankitk20) [inline goa/Classes/alphamon/Alphamon.cpp]: Put space before opening curly brace

File: goa/Classes/ext/LipiTKInterface.cpp [hunk 4]
```diff
@@ -101,7 +101,7 @@ std::map<std::string, std::string> LipiTKInterface::getEnglishUnicodeMapping() {
 std::map<std::string, std::string> LipiTKInterface::split(std::string s, char delim)
 {
     std::map<std::string, std::string> map;
-    std::vector<std::string> elems;
+    //std::vector<std::string> elems;
```
Comment (ankitk20) [inline goa/Classes/ext/LipiTKInterface.cpp:104]: Remove this

File: goa/Classes/ext/reco/shaperec/featureextractor/pointfloat/PointFloatShapeFeatureExtractor.cpp [hunk 5]
```diff
@@ -168,7 +168,7 @@ int PointFloatShapeFeatureExtractor::extractFeatures(const LTKTraceGroup& inTrac
         "PointFloatShapeFeatureExtractor::extractFeatures()" << endl;
     
     PointFloatShapeFeature *featurePtr = NULL;
-	float x,y,deltax;
+	//float x,y,deltax;
```
Comment (ankitk20) [inline goa/Classes/ext/reco/shaperec/featureextractor/pointfloat/PointFloatShapeFeatureExtractor.cpp:171]: Remove this

File: goa/Classes/ext/reco/shaperec/nn/NNShapeRecognizer.cpp [hunk 6]
```diff
@@ -4486,7 +4486,7 @@ int NNShapeRecognizer::processPrototypeSetForLVQ()
 
 	int codeVecIndex = 5;
 	int trainSize = m_trainSet.size() ;
-	int train_index = 0;
+	//int train_index = 0;
```
Comment (ankitk20) [inline goa/Classes/ext/reco/shaperec/nn/NNShapeRecognizer.cpp:4489]: Remove this line

File: goa/Classes/ext/reco/shaperec/preprocessing/LTKPreprocessor.cpp [hunk 7]
```diff
@@ -1237,9 +1237,9 @@ int LTKPreprocessor::dehookTraces(const LTKTraceGroup& inTraceGroup,
 
 			dPY3 = yVec[dominantPoints[1] + 1];
 
-			dPX4 = xVec[dominantPoints[1] - 1];
+			//dPX4 = xVec[dominantPoints[1] - 1];
```
Comment (ankitk20) [inline goa/Classes/ext/reco/shaperec/preprocessing/LTKPreprocessor.cpp:1240]: Remove this line

File: goa/Classes/ext/reco/shaperec/preprocessing/LTKPreprocessor.cpp [hunk 8]
```diff
@@ -1237,9 +1237,9 @@ int LTKPreprocessor::dehookTraces(const LTKTraceGroup& inTraceGroup,
...
 
-			dPY4 = yVec[dominantPoints[1] - 1];
+			//dPY4 = yVec[dominantPoints[1] - 1];
```
Comment (ankitk20) [inline goa/Classes/ext/reco/shaperec/preprocessing/LTKPreprocessor.cpp:1242]: Remove this line

File: goa/Classes/ext/util/lib/LTKCheckSumGenerate.cpp [hunk 9]
```diff
@@ -374,7 +374,7 @@ stringStringMap LTKCheckSumGenerate::updateHeaderWithMandatoryFields(const strin
 		 " Entering: LTKCheckSumGenerate::updateHeaderWithMandatoryFields()" << endl;
 
 	long testEndian = 1;
-	string comment="";
+	//string comment="";
```
Comment (ankitk20) [inline goa/Classes/ext/util/lib/LTKCheckSumGenerate.cpp:377]: Remove this line

File: goa/Classes/ext/reco/shaperec/preprocessing/LTKPreprocessor.cpp [hunk 10]
```diff
@@ -1273,9 +1273,9 @@ int LTKPreprocessor::dehookTraces(const LTKTraceGroup& inTraceGroup,
 
 			dPY3 = yVec[dominantPoints[numDominantPoints - 2] - 1];
 
-			dPX4 = xVec[dominantPoints[numDominantPoints - 2] + 1];
+			//dPX4 = xVec[dominantPoints[numDominantPoints - 2] + 1];
```
Comment (ankitk20) [inline goa/Classes/ext/reco/shaperec/preprocessing/LTKPreprocessor.cpp:1276]: Remove commented line

**Review by ankitk20 — CHANGES_REQUESTED** [review]

File: goa/Classes/alphamon/Alphamon.cpp (hunk 3 above)
Comment (jeevithiesh) [reply goa/Classes/alphamon/Alphamon.cpp]: I have added a space.

**Pull request closed without merging**